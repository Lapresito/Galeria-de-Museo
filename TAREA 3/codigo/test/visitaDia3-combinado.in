crearVisitaDia 01/01/2001
imprimirVisitaDia
liberarVisitaDia
crearVisitaDia 02/02/2002
crearGrupoABB 
crearVisitante 1 Nombre1 Apellido1 20
agregarAGrupoABB
encolarGrupoVisitaDia 
imprimirVisitaDia
crearGrupoABB 
crearVisitante 2 Nombre2 Apellido2 20
agregarAGrupoABB
encolarGrupoVisitaDia 
imprimirVisitaDia
cantidadGruposVisitaDia
desencolarGrupoVisitaDia
imprimirVisitaDia
crearGrupoABB 
crearVisitante 2 Nombre2 Apellido2 20
agregarAGrupoABB
encolarGrupoVisitaDia 
crearGrupoABB 
crearVisitante 3 Nombre3 Apellido3 20
agregarAGrupoABB
encolarGrupoVisitaDia 
crearGrupoABB 
crearVisitante 4 Nombre4 Apellido4 20
agregarAGrupoABB
encolarGrupoVisitaDia 
imprimirVisitaDia
desencolarGrupoVisitaDia
desencolarGrupoVisitaDia
imprimirVisitaDia
liberarVisitaDia
Fin