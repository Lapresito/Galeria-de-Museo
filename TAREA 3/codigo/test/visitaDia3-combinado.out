Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> La visita dia se ha creado de forma exitosa.
2> Visita para dia: 1/1/2001
3> La visita dia se ha liberado exitosamente.
4> La visita dia se ha creado de forma exitosa.
5> El grupoABB ha sido creado de forma exitosa.
6> Visitante creadox de forma exitosa.
7> Se ha agregado el visitante al grupo de forma exitosa.
8> Se ha agregado el grupo a la visita dia de forma exitosa.
9> Visita para dia: 2/2/2002
Visitante Nombre1 Apellido1
Id: 1
Edad: 20
10> El grupoABB ha sido creado de forma exitosa.
11> Visitante creadox de forma exitosa.
12> Se ha agregado el visitante al grupo de forma exitosa.
13> Se ha agregado el grupo a la visita dia de forma exitosa.
14> Visita para dia: 2/2/2002
Visitante Nombre2 Apellido2
Id: 2
Edad: 20
Visitante Nombre1 Apellido1
Id: 1
Edad: 20
15> La cantidad de grupos en la visita dia es 2
16> Visitante Nombre1 Apellido1
Id: 1
Edad: 20
17> Visita para dia: 2/2/2002
Visitante Nombre2 Apellido2
Id: 2
Edad: 20
18> El grupoABB ha sido creado de forma exitosa.
19> Visitante creadox de forma exitosa.
20> Se ha agregado el visitante al grupo de forma exitosa.
21> Se ha agregado el grupo a la visita dia de forma exitosa.
22> El grupoABB ha sido creado de forma exitosa.
23> Visitante creadox de forma exitosa.
24> Se ha agregado el visitante al grupo de forma exitosa.
25> Se ha agregado el grupo a la visita dia de forma exitosa.
26> El grupoABB ha sido creado de forma exitosa.
27> Visitante creadox de forma exitosa.
28> Se ha agregado el visitante al grupo de forma exitosa.
29> Se ha agregado el grupo a la visita dia de forma exitosa.
30> Visita para dia: 2/2/2002
Visitante Nombre4 Apellido4
Id: 4
Edad: 20
Visitante Nombre3 Apellido3
Id: 3
Edad: 20
Visitante Nombre2 Apellido2
Id: 2
Edad: 20
Visitante Nombre2 Apellido2
Id: 2
Edad: 20
31> Visitante Nombre2 Apellido2
Id: 2
Edad: 20
32> Visitante Nombre2 Apellido2
Id: 2
Edad: 20
33> Visita para dia: 2/2/2002
Visitante Nombre4 Apellido4
Id: 4
Edad: 20
Visitante Nombre3 Apellido3
Id: 3
Edad: 20
34> La visita dia se ha liberado exitosamente.
35> Fin.
